crates/flux-config/src/flags.rs

```diff
@@ -139,10 +139,16 @@ pub struct Paths {
 }
 
 impl Paths {
+    fn file_matches(p: &PathBuf, file: &str) -> bool {
+        let res = p.to_str().map_or(false, |p| p == file);
```

Comment:
I'm confused. How is this different than before? It is still checking for equality, no?